CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 91.1.0
+
+This minor release adds support for customizing elevation on ActionSheet component, and updates its
```

Comment:
...customizing _the_ elevation on...

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/8475/commits/63681a0943ba83b34456238526642274ef7a7842